### ISSUE TITLE
Getting Video but No Audio? 

### ISSUE THREAD

**Issue by CombatMedic02** [mkvd/smarttv-twitch#99]
Weird bug but lately when using the app I get video from stream but no audio. Anyone else having this issue? Did Twitch change something their end?

**Comment by jazha** [comment]
Yes I have the same thing on my TV. Youtube app works and sends audio, but the Twitch app doesn't.

**Comment by ghabit** [comment]
No sound at twitch too :(

**Comment by jazha** [comment]
@ghabit When you say "app reloading" do you mean just reloading the app or actually reinstalling it?

**Comment by ghabit** [comment]
I mean restarting the app. But, there are low chances to return the sound. Also it's possible to get the sound back by choosing quality manually. But yes, there is some problems with sound in the app.

**Comment by ghabit** [comment]
Update: I have not sound @ auto quality settings. The only working for me way to return the sound is to switch quality Tito 720.
